tests/Unit/Bridge/Symfony/DependencyInjection/ConfigurationTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\LoyaltyCorp\Search\Unit\Bridge\Symfony\DependencyInjection;
+
+use LoyaltyCorp\Search\Bridge\Symfony\DependencyInjection\Configuration;
+use Tests\LoyaltyCorp\Search\TestCases\UnitTestCase;
+
+/**
+ * @covers \LoyaltyCorp\Search\Bridge\Symfony\DependencyInjection\Configuration
+ */
+final class ConfigurationTest extends UnitTestCase
+{
+    /**
+     * Test getConfigTreeBuilder returns correct tree builder.
+     *
+     * @return void
+     */
+    public function testGetConfigTreeBuilder(): void
+    {
+        $treeBuilder = (new Configuration())->getConfigTreeBuilder();
+
+        self::assertArrayHasKey('use_listeners', $treeBuilder->getRootNode()->getChildNodeDefinitions());
```

Comment:
If the configuration is a required piece of functionality then there needs to be tests about what the nodes are, not just that they exist.